modules/ROOT/pages/introduction/index.adoc

```diff
@@ -27,6 +27,8 @@ This introduction will cover the following topics:
 * xref:introduction/cypher-aura.adoc[]
 
 
+include::{ga-promo-uri}/{ga-promo}.adoc[]
```

Comment:
There are now 2 mentions of GraphAcademy on this page.

Reply:
There should be more?

If it is a point of contention, I'm happy to remove the `Additionally...` sentence

Reply:
No, I think it was fine as it was.